Library/ViewModels/ThanksViewModelTests.swift

```diff
@@ -80,13 +80,17 @@ final class ThanksViewModelTests: TestCase {
 
       goToDiscovery.assertValues([.illustration])
       XCTAssertEqual(
-        ["Thanks Page Viewed"],
+        ["Page Viewed"],
         self.dataLakeTrackingClient.events
       )
       XCTAssertEqual(
-        ["Thanks Page Viewed"],
+        ["Page Viewed"],
         self.segmentTrackingClient.events
       )
+      XCTAssertEqual(
```

Comment:
Wherever we test the `segmentClient` we should usually test the `dataLakeClient` as well.